src/extensions/registries/page-menu-registry.ts

```diff
@@ -26,23 +26,20 @@ import type { PageTarget, RegisteredPage } from "./page-registry";
 import type { LensExtension } from "../lens-extension";
 import { BaseRegistry } from "./base-registry";
 
-export interface PageMenuRegistration {
-  target?: PageTarget;
-  title: React.ReactNode;
-  components: PageMenuComponents;
-}
-
-export interface ClusterPageMenuRegistration extends PageMenuRegistration {
+export interface ClusterPageMenuRegistration {
   id?: string;
   parentId?: string;
+  target?: PageTarget;
+  title: React.ReactNode;
+  components: ClusterPageMenuComponents;
 }
 
-export interface PageMenuComponents {
+export interface ClusterPageMenuComponents {
   Icon: React.ComponentType<IconProps>;
 }
 
-export class PageMenuRegistry<T extends PageMenuRegistration> extends BaseRegistry<T> {
-  add(items: T[], ext: LensExtension) {
+export class ClusterPageMenuRegistry extends BaseRegistry<ClusterPageMenuRegistration> {
```

Comment:
I consolidated this here because we no longer have `(Global)PageMenuRegistry` so we shouldn't leave this clutter around.